lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -684,16 +693,30 @@ static LogicalResult verifyParOp(ParOp parOp) {
   return success();
 }
 
-LogicalResult ParOp::canonicalize(ParOp parOp, PatternRewriter &rewriter) {
-  if (parOp.getBody()->empty()) {
+/// This pattern canonicalizes away a ParOp in cases where
+/// 1. it is nested within a seq structure
```

Comment:
Maybe just me, but I first interpreted this as either case needs to be true and this threw me off. Maybe throwing in and `and` will make this more explicit.